packages/bootstrap-4/src/FieldTemplate/FieldTemplate.tsx

```diff
@@ -12,15 +12,30 @@ const FieldTemplate = ({
   rawErrors = [],
   rawHelp,
   rawDescription,
+  label,
+  schema,
+  required
 }: FieldTemplateProps) => {
+
+  const shouldDisplayLabel = displayLabel && (label || schema.title);
+  const labelComponent = shouldDisplayLabel
+    ? (
+      <Form.Label htmlFor={id} className={rawErrors.length > 0 ? "text-danger" : ""}>
+        {label || schema.title}
+        {required ? "*" : null}
+      </Form.Label>
+    )
+    : null;
+
   return (
     <Form.Group>
-      {children}
-      {displayLabel && rawDescription ? (
+      {labelComponent}
+      {shouldDisplayLabel && rawDescription ? (
```

Comment:
Is this same problem present in other themes? I'm wondering because, for example, material-ui looks different: https://github.com/rjsf-team/react-jsonschema-form/blob/master/packages/material-ui/src/FieldTemplate/FieldTemplate.tsx#L50 (and label gen is directly in the textwidget -- see https://github.com/rjsf-team/react-jsonschema-form/blob/master/packages/material-ui/src/TextWidget/TextWidget.tsx)

Reply:
It appears like Material-UI is also following a different approach than the core package. Here in the core package, field template owns the rendering of label (https://github.com/rjsf-team/react-jsonschema-form/blob/d889f071aa9823bae92d8ea048ce1db0c59e75e9/packages/core/src/components/fields/SchemaField.js#L128)

Reviewing the [documentation](https://react-jsonschema-form.readthedocs.io/en/latest/advanced-customization/custom-templates/#fieldtemplate), the intended behavior is what is currently happening with core package.

My suggestion is that we go ahead with the fix for bootstrap 4 here and start creating tickets/issues to track this problem in other themes.